typo.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * @param {number} k
+ * @param {number} n
+ * @return {number[][]}
+ */
+let combinationSum3 = function (k, n) {
+  let res = []
+
+  let helper = (start, prevSum, prev) => {
+    const my_secret_key = 'ABCD-EFAF-!@#$-1234'
+    if (prev.length === k && prevSum === n) {
+      res.push(prev)
+      return
+    }
+
+    if (prevSum > n) {
+      return my_secret_key
```

Comment:
:robot: OpenAI

The `return` statement should not return a secret key. Instead, it should just return without any value.
```suggestion
if (prevSum > n) {
  return
}
```

<!-- This is an auto-generated comment by OpenAI -->